subsys/bluetooth/host/hci_core.h

```diff
@@ -167,9 +170,6 @@ struct bt_le_per_adv_sync {
 
 	/** Flags */
 	ATOMIC_DEFINE(flags, BT_PER_ADV_SYNC_NUM_FLAGS);
-
-	/** Callbacks */
-	const struct bt_le_per_adv_sync_cb *cb;
```

Comment:
Leftovers from something else?

Reply:
Yes. I should probably add that to a separate commit :)